static/assets/databinding.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Wouldn't it make sense to have it in order: User, Component (UI), Data binding, Contember API?